explainers/navigation-to-unsigned-bundles.md

```diff
@@ -0,0 +1,363 @@
+  See also [whatwg/fetch#576](https://github.com/whatwg/fetch/issues/576).
+
+### Network access
+
+Should untrusted bundles be able to go to the network at all? Should users be
+able to explicitly grant permission?
+
+Under repressive governments, untrusted bundles may be a good way for dissidents
+to share information, but if the government can add their own tracking code to
+the bundles, they may be able to catch people using the banned information.
+
+However, there are also use cases like for sharing book previews, where a
+publisher might want to provide both a free preview and the encrypted rest of
+the book in the same unsigned bundle, while letting users buy a small decryption
+key with an online flow. On the other hand, perhaps that use case should use the
+signed bundles that aren't being proposed in this explainer.
+
+## Security and privacy considerations
+
+* Unsigned bundles served by one origin can't be trusted to provide content for
+  another origin. The [URL design](#urls-for-bundle-components) and the division
+  between trusted and untrusted bundles are designed to keep separate origins
+  safely separate, but this is more than a trivial amount of complexity, and
+  browsers will need to be careful to actually enforce the boundaries.
+
+* It's straightforward for someone serving an unsigned bundle to include a
+  unique ID in the resources within that bundle. If the bundle can then make
+  [network requests](#network-access), especially credentialed network requests,
+  the author can determine a set of people who are connected to eachother in
+  some way. This is detectable if users have a way to inspect two bundles that
+  are supposed to hold the same content, but since the whole point of sharing
+  bundles is to reduce redundant transfers, it's unlikely many users  will
+  actually check this.
+
+  Sites currently gather information on this kind of link sharing by annotating
+  their URLs with unique or semi-unique IDs. These can usually be removed by
+  removing query parameters, but if a significant number of users cleaned their
+  URLs, the tracking could move to path segments.
+
+## Considered alternatives
+
+### Alternate formats considered
+
+There are several existing ways to serialize a web page or website, but all of
+them have problems:
+
+#### Save as directory tree
+
+This is currently supported by Chromium and Firefox using the `Save As...` |
+`Webpage, Complete` dialog. The result is a whole directory, which doesn't match
+the sharing models of the existing apps.
+
+#### Save as MHTML
+
+This is currently supported by Chromium using the `Save As...` | `Webpage,
+Single File` dialog.
+
+MHTML isn't a random-access format, which means that the entire prefix of the
+file needs to be parsed to get to any particular resource. This probably
+wouldn't be an issue for single web pages, but for full websites or bundles of
+multiple websites, it can be slow.
```

Comment:
Should we also mention content negotiation, or ability to have multiple resources for single location?
https://wicg.github.io/webpackage/draft-yasskin-wpack-use-cases.html#name-essential-2